api/imapsync/update

```diff
@@ -77,15 +77,15 @@ if ($cmd eq 'update') {
     my $exclude = join ('|',@{$input->{'Exclude'}}) || '';
     $exclude = '|'.$exclude  if ($exclude ne '');
     # write the Env file for systemd
-    my $vars = qq(
-    LOCALUSER=$input->{'name'}
+    my $vars = qq(    LOCALUSER=$input->{'name'}
```

Comment:
Whey the leading blanks?

```suggestion
    my $vars = qq(LOCALUSER=$input->{'name'}
```

Reply:
With this fix, the file still has spaces at the begging of each line: it's just a consequence of code indention.